<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="TestNGSessions.LoginPageTest" tests="1" time="26.873" ignored="2" failures="1" timestamp="30 Oct 2019 00:29:41 GMT" skipped="0" hostname="savvy" errors="0">
  <testcase name="loginTest" time="26.873" classname="TestNGSessions.LoginPageTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	header text is not matched... expected [Sales Dashboard123] but found [Sales Dashboard],
	expected [Reports dashboard123] but found [Reports dashboard]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	header text is not matched... expected [Sales Dashboard123] but found [Sales Dashboard],
	expected [Reports dashboard123] but found [Reports dashboard]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at TestNGSessions.LoginPageTest.loginTest(LoginPageTest.java:69)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginPageTitleTest" classname="TestNGSessions.LoginPageTest">
    <skipped/>
  </testcase> <!-- loginPageTitleTest -->
  <testcase name="signUpLinkTest" classname="TestNGSessions.LoginPageTest">
    <skipped/>
  </testcase> <!-- signUpLinkTest -->
</testsuite> <!-- TestNGSessions.LoginPageTest -->
